From 26c96a730ddc423964be16cd92175039c0d4181e Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Wed, 29 Mar 2017 14:30:07 +0800
Subject: [PATCH 1623/5242] MLK-14544 arm: dts: imx7ulp: correct earlycon
 port.membase address

commit  90110f2499c2b69380307485e368d08086fe67cd from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct i.MX7ulp earlycon port.membase address for arm2 and evk
board dts file.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-14x14-arm2.dts |    2 +-
 arch/arm/boot/dts/imx7ulp-evk.dts        |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-14x14-arm2.dts b/arch/arm/boot/dts/imx7ulp-14x14-arm2.dts
index 5278cb8..8a67474 100644
--- a/arch/arm/boot/dts/imx7ulp-14x14-arm2.dts
+++ b/arch/arm/boot/dts/imx7ulp-14x14-arm2.dts
@@ -15,7 +15,7 @@
 	compatible = "fsl,imx7ulp-14x14-arm2", "fsl,imx7ulp", "Generic DT based system";
 
 	chosen {
-		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x402D0010,115200";
+		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x402D0000,115200";
 		stdout-path = &lpuart4;
 	};
 
diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 4e34092..e616bf4 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -16,7 +16,7 @@
 	compatible = "fsl,imx7ulp-evk", "fsl,imx7ulp", "Generic DT based system";
 
 	chosen {
-		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x402D0010,115200";
+		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x402D0000,115200";
 		stdout-path = &lpuart4;
 	};
 
-- 
1.7.9.5

